### ISSUE TITLE
Things aren't documented!

### ISSUE THREAD

**Issue by coderstephen** [coderstephen/slack-client#2]
This is an issue to remind myself to set up documentation on how to use stuff :P.

**Comment by pavel-voronin** [comment]
Yes, please!

**Comment by thetylerhayes** [comment]
This lib is amazing but yeah motion very much seconded for more docs :+1: :+1: I've spent a couple hours now today digging around the code to get everything working

**Comment by coderstephen** [comment]
Yeah, and I feel bad for it, as I should. :disappointed: This library just isn't high enough priority on my OSS project list.

I'm a little hesitant to write documentation right now because of #10; I plan on rewriting the core of the library within the month, which may include some significant API changes.
